### DISCUSSION TITLE
Have an option to allow the email to look like it has sent from the sender instead of AnonAddy

### DISCUSSION THREAD

**Discussion by VideoScape** [anonaddy/anonaddy#194] [Feature Requests]
Hey so the reasoning for this is so I can improve my spam filtering on my domain side.

So for example add a setting that says the email is sent from bob@github.com instead via github@anonaddy.com in the "From " field as it seems my spam filtering is getting confused by the custom domains I am using and allowing more than expected.

So from a user's perspective, the only thing the user knows it was routed through AnonAddy was the banner at the top. 

Hope that makes sense!

Thanks!

**Comment by willbrowningme** [comment]
I'm afraid this is not possible with AnonAddy since emails are forwarded on as an entirely new message.

If I were to leave the "From:" header as bob@github.com then it would be classed as an attempt to spoof GitHub's domain and would fail DMARC checks which would result in the message going to spam or being rejected outright.

The AnonAddy server cannot spoof or send emails from other domains, which is why the "From:" header has to be from the alias itself.
